File: bubble_sort.php
<?php

function bubble_sort(&$mang)
{
	$phan_tu = sizeof($mang);
	for($i = 0; $i < $phan_tu - 1; $i++)
	{
		for ($j = $phan_tu - 1; $j > $i - 1; $j--)
		{
			if ($mang[$j] < $mang[$j-1])
			{
				$tmp = $mang[$j];
				$mang[$j] = $mang[$j-1];
				$mang[$j-1] = $tmp;
			}
		}
	}
}

$mang = array(3, 5, 6, 1, 2, 0, 8, 9, 4, 7);
bubble_sort($mang);
for ($i = 0; $i < sizeof($mang); $i++)
	echo $mang[$i]." ";

?>

File: bubble_sort.py
from typing import List


def bubble_sort(mang: List[int]) -> None:
    phan_tu = len(mang)
    for i in range(phan_tu):
        for j in range(phan_tu - 1, i, -1):
            if mang[j] < mang[j-1]:
                mang[j], mang[j-1] = mang[j-1], mang[j]

if __name__ == "__main__":
    mang = [3, 5, 6, 1, 2, 0, 8, 9, 4, 7]
    bubble_sort(mang)
    print(mang)
